src/lammpsio/data.py

```diff
@@ -394,6 +482,62 @@ def read(self):
                                 "Expected number of columns not read for mass"
                             )
                         masses[int(row[0])] = float(row[1])
+                elif "Bonds" in line:
+                    if N_bonds is not None:
+                        snap.bonds = Bonds(N_bonds)
+                    _readline(f, True)  # blank line
+                    for i in range(int(snap.bonds.N)):
+                        row = _readline(f, True).split()
+                        if len(row) < 4:
+                            raise IOError(
+                                "Expected number of columns not read for bonds"
+                            )
+                        row = [int(x) for x in row]
+                        snap.bonds.id[i] = row[0]
+                        snap.bonds.typeid[i] = row[1]
+                        snap.bonds.members[i] = row[2:]
```

Comment:
Explicitly slice the columns you want
```suggestion
                        snap.bonds.members[i] = row[2:4]
```